ch2/Makefile

```
@@ -0,0 +1,3 @@
+init:
+	pip install -U pip
+	pip install -r requirements.txt
```

```suggestion
init:
	pip install -U pip
	pip install -r requirements.txt

server:
	docker-compose up -d

server-clean:
	docker-compose down -v
	docker rmi ch2-mlflow
```
@Kimdongui 동현님 추가적으로 `docker-compose up` 를 사용할 때 -d 옵션을 사용해서 백그라운드로 실행시키도록 만드는게 좋을 것 같구요. `docker-compose down` 할 때도 -v 옵션을 줘서 잡힌 volume 들을 다 지우는게 좋을 것 같아요. 그리고 down할 때 관련 이미지도 지우도록 하는게 좋을 것 같습니다.

---

@dongminlee94  동민님 저도 동민님 PR 보고 `server-clean` 부분을 똑같은 형태로 작성을 했는데요. 해보다 보니 docker compose 로 dockerfile 을 build 할 때 자동으로 생기는 이미지 형태가 `ch2-mlflow` 가 아니라 `ch2_mlflow`여야 하지 않나요?
제 로컬에서는 그렇게 되는데 혹시 다른가 해서 여쭤봅니다!

---

네네 이부분도 맞추겠습니다~

---

@datawhales 저는 `-` 였어요. 혹시 `_` 셧나요?